### PR TITLE
Add team information to BambooHR converter

### PR DESCRIPTION
**Pull request by cjwooo**
## Description

> Provide description here

## Type of change
- [ ] Bug fix
- [x] New feature
- [ ] Breaking change

## Related issues

> Fix [#1]() 

## Migration notes

> Describe migration notes if any

## Extra info

> Add any additional information

### CODE REVIEW

File: destinations/airbyte-faros-destination/src/converters/bamboohr/users.ts [hunk 1]
```diff
@@ -93,4 +114,57 @@ export class Users extends BambooHRConverter {
...
+    )) {
+      const parentTeamId = this.managers.get(uid);
+      res.push({
+        model: 'org_Team',
+        record: {
+          uid,
+          name: `${this.employeeIdsToNames.get(uid)} Org`,
+          lead: {uid},
+          parentTeam: parentTeamId ? {uid: parentTeamId} : null,
+          teamChain: this.computeManagerChain(uid),
```
Comment (tovbinm) [inline destinations/airbyte-faros-destination/src/converters/bamboohr/users.ts:136]: We dont need to compute `teamChain` anymore, unless you want to have it in Faros CE?

**Review by tovbinm — APPROVED** [review]
lgtm!

File: destinations/airbyte-faros-destination/src/converters/bamboohr/users.ts (hunk 1 above)
Comment (cjwooo) [reply destinations/airbyte-faros-destination/src/converters/bamboohr/users.ts:136]: I don't know whether or not it's used in CE, if not, I'll take it out. @ypc-faros @thomas-gerber

Comment (tovbinm) [reply destinations/airbyte-faros-destination/src/converters/bamboohr/users.ts:136]: Not used. We dont have `org` namespace in Faros CE.

**Pull request merged**